app/views/products/all_products.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <%= render 'layouts/navbar' %>
+<h2>All Products</h2>
+<% @products.each_with_index do |product, index| %>
+  <% if product.status == "Active" && product.quantity != 0 %>
+    <% if index % 6 == 0 %>
```

Comment:
I think this section of the code is intended to put 6 product cards in each visual row. It's fantastic that you're trying to control the appearance of each visual row on the page! Unfortunately, inspecting the page reveals that each row winds up getting nested inside the row(s) that came before it. Also, using `index % 6` to split these probably isn't the best way. I'd probably use bootstrap cols without the rows. Each card would also have to be uniform size (width and height) for this to work.